misc/flatpak/org.qutebrowser.qutebrowser.json

```diff
@@ -0,0 +1,197 @@
+        {
+            "name": "cssutils",
+            "buildsystem": "simple",
+            "build-commands": ["python3 setup.py install --prefix=/app"],
+            "ensure-writable": ["easy-install.pth"],
+            "cleanup": [ "/bin" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/5c/0b/c5f29d29c037e97043770b5e7c740b6252993e4b57f029b3cd03c78ddfec/cssutils-1.0.2.tar.gz",
+                    "sha256": "a2fcf06467553038e98fea9cfe36af2bf14063eb147a70958cfcaa8f5786acaf"
+                }
+            ]
+        },
+        {
+            "name": "Jinja2",
+            "buildsystem": "simple",
+            "build-commands": ["python3 setup.py install --prefix=/app"],
+            "ensure-writable": ["easy-install.pth"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/56/e6/332789f295cf22308386cf5bbd1f4e00ed11484299c5d7383378cf48ba47/Jinja2-2.10.tar.gz",
+                    "sha256": "f84be1bb0040caca4cea721fcbbbbd61f9be9464ca236387158b0feea01914a4"
+                }
+            ]
+        },
+        {
+            "name": "Pygments",
+            "buildsystem": "simple",
+            "build-commands": ["python3 setup.py install --prefix=/app"],
+            "ensure-writable": ["easy-install.pth"],
+            "cleanup": [ "/bin" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/71/2a/2e4e77803a8bd6408a2903340ac498cb0a2181811af7c9ec92cb70b0308a/Pygments-2.2.0.tar.gz",
+                    "sha256": "dbae1046def0efb574852fab9e90209b23f556367b5a320c0bcb871c77c3e8cc"
+                }
+            ]
+        },
+        {
+            "name": "pyPEG2",
+            "buildsystem": "simple",
+            "build-commands": ["python3 setup.py install --prefix=/app"],
+            "ensure-writable": ["easy-install.pth"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/f9/bd/10398e2c2d2070cc8a9c7153abfbd4ddb2895a2c52a32722ab8689e0cc7d/pyPEG2-2.15.2.tar.gz",
+                    "sha256": "2b2d4f80d8e1a9370b2a91f4a25f4abf7f69b85c8da84cd23ec36451958a1f6d"
+                }
+            ]
+        },
+        {
+            "name": "PyYAML",
+            "buildsystem": "simple",
+            "build-commands": ["python3 setup.py install --prefix=/app"],
+            "ensure-writable": ["easy-install.pth"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://pyyaml.org/download/pyyaml/PyYAML-3.12.tar.gz",
+                    "sha256": "592766c6303207a20efc445587778322d7f73b161bd994f227adaa341ba212ab"
+                }
+            ]
+        },
+        {
+            "name": "PyQT5",
```

Comment:
Why build PyQt5 and sip from source instead of using the wheels (pre-built binaries) from PyPI?

Reply:
I will have a look at it and try rebasing the flatpak on the org.freedesktop.Sdk.  Currently the flatpak has got a download size of ~10 MByte. The wheel package only has got a size of 100MByte. So the size would increase drastically.